### PR TITLE
add onImagesLoaded props

### PR DESCRIPTION
**Pull request by tonytonyjan**
we might need this to solve some problems like this one: https://css-tricks.com/forums/topic/masonry-loading-images-displaynone-issue/

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -25,12 +25,14 @@ var MasonryComponent = React.createClass({
...
     propTypes: {
         disableImagesLoaded: React.PropTypes.bool,
+        onImagesLoaded: React.PropTypes.func,
         options: React.PropTypes.object
     },
 
     getDefaultProps: function() {
         return {
             disableImagesLoaded: false,
+            onImagesLoaded: _ => _,
```
Comment (afram) [inline lib/index.js]: We can't use arrow functions in source just yet as we need to support older versions of node at this moment. 

If we setup a build step then sure, but right now we don't have one.

**Comment by afram** [comment]
@tonytonyjan Thanks for your contribution!

Do you mind please adding a section in the Readme about this new handler as well as updating the tests?

File: lib/index.js (hunk 1 above)
Comment (tonytonyjan) [inline lib/index.js]: Thanks for the feedback, I have made some changes for compatibility.

**Comment by afram** [comment]
@tonytonyjan cheers for the changes. Are you planning on adding tests for your new functionality?

**Comment by tonytonyjan** [comment]
@afram I am not sure, I have never written test for reactjs before, I can try if I am available next week.

**Pull request merged**